### PR TITLE
Rc66 commerce workaround for broken qt5.10

### PR DESCRIPTION
**Pull request by howard-stearns**
RC66 form of https://github.com/highfidelity/hifi/pull/12772
See that for notes, test plan, etc.

### CODE REVIEW

File: interface/resources/qml/controls/FlickableWebViewCore.qml [hunk 1]
```diff
@@ -122,9 +122,21 @@ Item {
             newViewRequestedCallback(request)
         }
 
+        // Prior to 5.10, the WebEngineView loading property is true during initial page loading and then stays false
+        // as in-page javascript adds more html content. However, in 5.10 there is a bug such that adding html turns
+        // loading true, and never turns it false again. safeLoading provides a workaround, but it should be removed
+        // when QT fixes this.
```
Comment (jherico) [inline interface/resources/qml/controls/FlickableWebViewCore.qml]: can we reference the Qt bug here?  Like provide a link to the confirmed open bug?

Comment (howard-stearns) [reply interface/resources/qml/controls/FlickableWebViewCore.qml]: Good idea. The person who files that with QT should do so. :-)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8460/HighFidelity-Beta-PR12774-01d90ea1a5c50a266d6c003343d5895e33805ff3.dmg

File: interface/resources/qml/controls/FlickableWebViewCore.qml (hunk 1 above)
Comment (howard-stearns) [reply interface/resources/qml/controls/FlickableWebViewCore.qml]: you do it!

Comment (howard-stearns) [reply interface/resources/qml/controls/FlickableWebViewCore.qml]: did you even look at the ticket?

**Comment by sethalves** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8483/HighFidelity-Beta-PR12774-01d90ea1a5c50a266d6c003343d5895e33805ff3.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8483/HighFidelity-Beta-PR12774-01d90ea1a5c50a266d6c003343d5895e33805ff3.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Marketplace categories work as expected, same with Snap and Help apps.

**Pull request merged**